tools/update_kernel_commits.py

```diff
@@ -0,0 +1,599 @@
+#!/usr/bin/env python3
+"""
+Copyright (c) 2023 Zededa, Inc.
+SPDX-License-Identifier: Apache-2.0
+
+This script does two things:
+1. Generates the kernel-commits.mk-new file with commit information from GitHub branches.
+2. Generates a commit message with commit subjects and corresponding commit hashes between
+    old and new commits for each branch.
+
+this script queries the GitHub API to get the latest commit hashes for the kernel branches
+The generated commit message is saved to kernel-update-commit-message.txt
+
+Note:
+- This module depends on the 'requests' library for making HTTP requests to the GitHub API.
+- Make sure to review the generated commit message before committing the changes to your repository.
+
+Author: Mikhail Malyshev <mike.malyshev@gmail.com>
+"""
+
+import argparse
+import datetime
+import json
+import os
+import re
+import requests
+from colorama import Fore, Style, init
```

Comment:
Please, use
```python
from colorama import Fore, Style, init #pylint: disable=E0401
```
To remove yetus error until a better solution is provided (include colorama module in Yetus' container?)